setup.py

```diff
@@ -61,6 +61,7 @@
                       'configobj',
                       'six >= 1.9', # python-openstackclient won't work properly with less
                       'httplib2',
+                      'nose',
```

Comment:
I don't *love* the idea of adding a dependency for just one task, but I don't have a better solution yet. In the meantime could you add a comment explaining what this new dep is for? I don't want the reason to get buried in commit history.